lib/msprime.c

```diff
@@ -6590,7 +6610,9 @@ msp_beta_common_ancestor_event(msp_t *self, population_id_t pop_id, label_id_t l
 static double
 genic_selection_stochastic_forwards(double dt, double freq, double alpha, double u)
 {
-    double ux = (alpha * freq * (1 - freq)) / tanh(alpha * freq);
+    /* this is scaled following Ewens chapter 5 e.g.,
+     * w_11=1+s; w_12=1+s/2; w_22=1; that is h=0.5 */
+    double ux = (alpha / 2.0 * freq * (1 - freq)) / tanh(alpha / 2.0 * freq);
```

Comment:
I can never remember the precedence for things like ``x / y * z``. Would you mind putting in some brackets, or moving  the division to the end of the expression please?